js/classes/entities/midground/chad.js

```diff
@@ -188,7 +188,7 @@ class Chad {
             this.health -= amount;
             if (this.health <= 0) {
                 // Chad should die here
-                ASSET_MGR.playAudio(SFX.GAME_OVER.path, SFX.GAME_OVER.volume);
+                ASSET_MGR.playSFX(SFX.GAME_OVER.path, SFX.GAME_OVER.volume);
```

Comment:
Thanks for making this change. Any ideas what was wrong with playAudio being used here?

Reply:
`playAudio` no longer exists; it was replaced by `playSFX`.